libs/domain/picking/src/components/picking-product-card/picking-product-card.component.ts

```diff
@@ -0,0 +1,184 @@
+import { i18n } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { property, state } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import {
+  EVENT_EDIT,
+  EVENT_SUBMIT,
+  ItemsFilters,
+  PickingListItem,
+  ProductItemPickedEvent,
+  SummaryInfo,
+} from '../../models';
+import { styles } from './picking-product-card.styles';
+
+export class PickingProductCardComponent extends LitElement {
```

Comment:
Let's have stories for new components

Reply:
If stories is necessary for this case let's add a sub-task or an independent task fot that, let's not deal with that on this pr